splitio/sync/split.py

```diff
@@ -283,12 +283,12 @@ def _read_splits_from_legacy_file(cls, filename):
     @classmethod
     def _read_splits_from_yaml_file(cls, filename):
         """
-        Parse a splits file and return a populated storage.
+        Parse a feature flags file and return a populated storage.
 
-        :param filename: Path of the file containing mocked splits & treatments.
+        :param filename: Path of the file containing mocked feature flags & treatments.
         :type filename: str.
 
-        :return: Storage populataed with splits ready to be evaluated.
+        :return: Storage populataed with feature flags ready to be evaluated.
```

Comment:
Let's take advantage of this to fix the typo 😝 
```suggestion
        :return: Storage populated with feature flags ready to be evaluated.
```